device/flasher.go

```diff
@@ -90,45 +90,62 @@ func (d *flasher) PrepareForFlashing() error {
 		time.Sleep(time.Second)
 	}
 
-	fmt.Println("[+] Starting download", d.device)
-	log.WithField("url", d.devRepo.Image.URL).WithField("dir", d.devRepo.Dir()).Debug("download")
-	zipName, bar, err := help.DownloadFromUrlWithAttemptsAsync(d.devRepo.Image.URL, d.devRepo.Dir(), 3, wg)
-	if err != nil {
-		return err
-	}
-
-	bar.Prefix(fmt.Sprintf("[+] Download %-15s", zipName))
-	bar.Start()
-	wg.Wait()
-	bar.Finish()
-	time.Sleep(time.Second * 2)
-
 	err = help.DeleteHost(filepath.Join((os.Getenv("HOME")), ".ssh", "known_hosts"), "localhost")
 	if err != nil {
 		log.Error(err)
 	}
+	fileName := ""
```

Comment:
__[gometalinter]__
* [ineffassign] ineffectual assignment to fileName

<!-- Comment by [SideCI](https://sideci.com) -->